### DISCUSSION TITLE
#712 (Ascii Healthhud) feedback issue

### DISCUSSION THREAD

**Discussion by mcmeiler** [Bubberstation/Bubberstation#723] [codebase discussion]
Feedback issue for #712 in ~~accordance with the [contribution rules](https://github.com/Bubberstation/Bubberstation/pull/682).~~ Nevimer moved this to a discussion. Nice to know that the rules aren't relevant anymore.

I _highly doubt_ this will actually get looked at, however.
___
**This should be a preference if anything.**
Forcing people, if they play medical, to look at fugly numbers/4 bars with different colors 24/7 isn't a good idea. I'd rather have a single bar and examine them to see the type of their injury - the bar, like an examine, gives you an estimate on how badly injured they are.

<details>
<summary>Why I think this is unnecessary from personal in-game experience</summary>
E.g.  Person with a nearly red health bar comes in, quick shift click to see what's wrong.
Severe bruising? Brute.
Severe burns? Burn.
Nothing visible? Toxin, or oxygen, though at that point they'd be unconscious.
It's really not that hard, and doesn't require changing the HUD for everyone in my opinion.
</details>

Good thing that I already don't wear medHUDs as medical because they take up the slot I use for cosmetic glasses.

**Comment by SovietSpaghetti** [comment]
Yeah, this is just ugly. Echoing what was said above, you can always just examine people. If you need exact numbers to know the doses to give them? Scan them with the analyzer. I don't really see the need of this, an extra half a second from examining? At least you could turn off the wacky item outlines from earlier, but this is an eyesore. 

It also looks wacky with emissives and runechat

**Reply by mcmeiler** [reply]
Good to know I'm not the only one who thinks this way. Too bad both our words will fall on deaf ears.

**Comment by YakumoChen** [comment]
Looks really bad, and higher numbers take up way too much screen real estate. Needs to be a preference or a toggle (that defaults to off)
![image](https://github.com/Bubberstation/Bubberstation/assets/10399117/5585532a-5bf6-4ed5-81a9-84b94fe68f85)
